2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Configure stats pid to 132598
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/dolphin_classification/notebooks/wandb/settings
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230811_035718-ree9pbnl/logs/debug.log
2023-08-11 03:57:18,473 INFO    MainThread:132598 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230811_035718-ree9pbnl/logs/debug-internal.log
2023-08-11 03:57:18,474 INFO    MainThread:132598 [wandb_init.py:init():547] calling init triggers
2023-08-11 03:57:18,474 INFO    MainThread:132598 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'arch': 'InceptionV4'}
2023-08-11 03:57:18,474 INFO    MainThread:132598 [wandb_init.py:init():571] re-initializing run, found existing run on stack: nfssv5ul
2023-08-11 03:57:18,477 INFO    MainThread:132598 [wandb_run.py:_finish():1894] finishing run yashchks87/dolphin/nfssv5ul
2023-08-11 03:57:18,478 INFO    MainThread:132598 [jupyter.py:save_history():445] not saving jupyter history
2023-08-11 03:57:18,478 INFO    MainThread:132598 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-08-11 03:57:18,478 INFO    MainThread:132598 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-08-11 03:57:18,478 INFO    MainThread:132598 [wandb_run.py:_atexit_cleanup():2128] got exitcode: 0
2023-08-11 03:57:18,478 INFO    MainThread:132598 [wandb_run.py:_restore():2111] restore
2023-08-11 03:57:18,478 INFO    MainThread:132598 [wandb_run.py:_restore():2117] restore done
2023-08-11 03:57:22,374 INFO    MainThread:132598 [wandb_run.py:_footer_history_summary_info():3481] rendering history
2023-08-11 03:57:22,375 INFO    MainThread:132598 [wandb_run.py:_footer_history_summary_info():3513] rendering summary
2023-08-11 03:57:22,381 INFO    MainThread:132598 [wandb_run.py:_footer_sync_info():3440] logging synced files
2023-08-11 03:57:22,491 INFO    MainThread:132598 [wandb_init.py:init():596] starting backend
2023-08-11 03:57:22,491 INFO    MainThread:132598 [wandb_init.py:init():600] setting up manager
2023-08-11 03:57:22,495 INFO    MainThread:132598 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-08-11 03:57:22,496 INFO    MainThread:132598 [wandb_init.py:init():606] backend started and connected
2023-08-11 03:57:22,528 INFO    MainThread:132598 [wandb_run.py:_label_probe_notebook():1234] probe notebook
2023-08-11 03:57:22,529 INFO    MainThread:132598 [wandb_run.py:_label_probe_notebook():1244] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-08-11 03:57:22,529 INFO    MainThread:132598 [wandb_init.py:init():697] updated telemetry
2023-08-11 03:57:22,727 INFO    MainThread:132598 [wandb_init.py:init():730] communicating run to backend with 60.0 second timeout
2023-08-11 03:57:23,801 INFO    MainThread:132598 [wandb_run.py:_on_init():2180] communicating current version
2023-08-11 03:57:23,894 INFO    MainThread:132598 [wandb_run.py:_on_init():2189] got version response 
2023-08-11 03:57:23,894 INFO    MainThread:132598 [wandb_init.py:init():781] starting run threads in backend
2023-08-11 03:57:24,002 INFO    MainThread:132598 [wandb_run.py:_console_start():2159] atexit reg
2023-08-11 03:57:24,002 INFO    MainThread:132598 [wandb_run.py:_redirect():2014] redirect: wrap_raw
2023-08-11 03:57:24,002 INFO    MainThread:132598 [wandb_run.py:_redirect():2079] Wrapping output streams.
2023-08-11 03:57:24,003 INFO    MainThread:132598 [wandb_run.py:_redirect():2104] Redirects installed.
2023-08-11 03:57:24,007 INFO    MainThread:132598 [wandb_init.py:init():822] run started, returning control to user process
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Current SDK version is 0.15.8
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Configure stats pid to 132598
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/dolphin_classification/notebooks/wandb/settings
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_init.py:_log_setup():507] Logging user logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230811_040024-bmr9bxsx/logs/debug.log
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_init.py:_log_setup():508] Logging internal logs to /home/ubuntu/dolphin_classification/notebooks/wandb/run-20230811_040024-bmr9bxsx/logs/debug-internal.log
2023-08-11 04:00:24,414 INFO    MainThread:132598 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f4eeb75d100>
2023-08-11 04:00:24,415 INFO    MainThread:132598 [wandb_init.py:init():547] calling init triggers
2023-08-11 04:00:24,415 INFO    MainThread:132598 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'arch': 'InceptionV4'}
2023-08-11 04:00:24,415 INFO    MainThread:132598 [wandb_init.py:init():571] re-initializing run, found existing run on stack: ree9pbnl
2023-08-11 04:00:24,418 INFO    MainThread:132598 [wandb_run.py:_finish():1894] finishing run yashchks87/dolphin/ree9pbnl
2023-08-11 04:00:24,418 INFO    MainThread:132598 [wandb_run.py:_atexit_cleanup():2128] got exitcode: 0
2023-08-11 04:00:24,418 INFO    MainThread:132598 [wandb_run.py:_restore():2111] restore
2023-08-11 04:00:24,418 INFO    MainThread:132598 [wandb_run.py:_restore():2117] restore done
2023-08-11 04:00:28,934 INFO    MainThread:132598 [wandb_run.py:_footer_history_summary_info():3481] rendering history
2023-08-11 04:00:28,934 INFO    MainThread:132598 [wandb_run.py:_footer_history_summary_info():3513] rendering summary
2023-08-11 04:00:28,941 INFO    MainThread:132598 [wandb_run.py:_footer_sync_info():3440] logging synced files
